include/core/CCompressedLfuCache.h

```diff
@@ -161,7 +161,7 @@ class CCompressedLfuCache {
                         readValue(value);
                         return;
                     }
-                    m_LostCount += lastEvictedCount;
+                    m_RemovedCount += lastEvictedCount;
```

Comment:
I shifted this loss of count from `lost` to `removed` as it's similar to what happens in a resize.